client/main.client.js

```diff
@@ -82,3 +82,18 @@ Meteor.startup(() => {
         return pathWithTrailingSlash;
     });
 });
+
+window.onbeforeunload = function (e) {
+    if(Meteor.status().connected) {
+        e.cancel();
+    }
+    const message = "Do you really want to leave 4Minitz?",
+        e = e || window.event;
```

Comment:
Please use another name for this variable, e.g.
```event = e || window.event;```

Adjust the usage of `e` inside the `if` condition accordingly.